java-rest-service/accelerator.yaml

```diff
@@ -72,14 +82,15 @@ accelerator:
         name: exposeOpenAPIEndpoint
   imports:
   - name: java-version
+  - name: spring-boot-postgresql
+  - name: spring-boot-mysql
```

Comment:
Looks like the `spring-boot-mysql` options are conflicting with the ones in `spring-boot-postgresql` since they use the same name

```
- choices:
  - text: PostgreSQL
    value: postgres
  - text: MySQL
    value: mysql
  defaultValue: postgres
  inputType: select
  label: Database to use
  name: databaseType
  required: true
...
- defaultValue: postgres-database
  description: The name of the database instance this application will use (can be
    existing one in the cluster)
  inputType: text
  label: Database Instance Name
  name: databaseName
  required: true
...
- choices:
  - text: Testcontainers leveraging Docker (MySQL)
    value: testcontainers
  - text: Java In-Memory Database (H2)
    value: in-memory
  defaultValue: testcontainers
  description: In order to test database interaction you can choose between testcontainers
    (https://www.testcontainers.org/) (Docker with MySQL) or in-memory database (H2).
    Preferable use testcontainers as that is providing the target database technology.
  inputType: select
  label: Use testcontainers or in-memory database for integration testing
  name: databaseIntegrationTestType
  required: true
```

Reply:
Solved (by using more generic value and description).